gits/test/capabilities_test.py

```diff
@@ -300,18 +255,13 @@ def test_cap_cup(self):
         """
         term = self._terminal
 
-        # The cursor is at the left-most position.
-        # Note that the y and x values start from 1.
```

Comment:
I know that there's a corresponding comment in `_check_cap_cup`, but still it's quite useful.